src/tests/macros.ts

```diff
@@ -87,7 +87,7 @@ describe('Macros', () => {
     it('@alias works with arbitrary classes that implement get and set', () => {
       class SomeClass {
         inner = {
-          bar: 'baz'
+          bar: 'baz',
```

Comment:
Running `yarn lint` found this issue, probably because the linter and tests are not run automatically in this repo.